docs/process.rst

```diff
@@ -1674,6 +1674,19 @@ in the configuration file to select and configure subset of processes having sim
 
 See the :ref:`config-process-selectors` documentation for details.
 
+The ``label`` directive can be also expressed as a `Map<key-value>` or a `key=value` sentence:
```

Comment:
This now has been implemented with a new `resourceLabels` directive. Please update the docs

Reply:
@bentsherman can you take a look to (not sure if it's well expressed) https://github.com/nextflow-io/nextflow/pull/2853/files#diff-8b2232009593fccf012f8206c47ff7514a2fb5e4f98d190bbfafebbf36078c8fR1677